website/src/content/api-reference/application-config.mdx

```diff
@@ -355,6 +357,68 @@ Using `manage_` OAuth Scopes always imply the corresponding `view_` OAuth Scope.
 
 </Info>
 
+## `additionalOAuthScopes`
+
+The configuration for [additional OAuth Scopes and user permissions](/concepts/oauth-scopes-and-user-permissions#additional-oauth-scopes) extending [oAuthScopes configuration](#oauthscopes).
+
+## `additionalOAuthScopes.name`
+
+A name used to build the permission keys based on the default permissions pair.
+
+The `additionalOAuthScopes.name` value must adhere to the following restrictions:
```

Comment:
Let's try to use the same terminology "permission group" to make it easier for users to relate things to each other.

````suggestion
A unique name for the additional permission group.

```json
{
  "additionalOAuthScopes": [
    {
      "name": "movies",
    }
  ]
}
```

The name value must adhere to the following restrictions:
````